server/models/boxCrime.js

```diff
@@ -0,0 +1,104 @@
+var db = require('../../database/index').db;
+var BoxCrime = require('../../database/models/BoxCrime');
+
+var storeOpenDataInBoxes = (crimeData) => {
+  
+  var asyncBoxStore = crimeData.forEach((crime) => {
```

Comment:
After doing some testing in the console, forEach does not return anything. Therefore asyncBoxStore will always be undefined. The way I got around this is that I created a promises array outside of the forEach and pushed to that array inside the forEach.

check my test code out here: 
http://jsbin.com/fihimimuyi/edit?js,console